src/thing/Portal.cpp

```diff
@@ -109,32 +109,24 @@ void Portal::Start() {
   });
 
   server_.on("/config", [&] () {
-    DynamicJsonBuffer jsonBuffer;
     if (server_.method() == HTTP_GET) {
-      JsonObject& root = jsonBuffer.createObject();
-      root["host"] = config_.host.c_str();
-      root["auth"] = config_.auth.c_str();
-      root["path"] = config_.path.c_str();
-      root["wifi_ssid"] = config_.wifi_ssid.c_str();
-      root["wifi_key"] = config_.wifi_key.c_str();
+      auto client = server_.client();
```

Comment:
should we also write `200 OK` to the client and some headers to be a valid HTTP response?